Introducing: Minimum Viable Product Marketing (#16809)
PostHog/posthog.com: contents/blog/the-dos-and-donts-of-minimum-viable-product-marketing.md

---
title: "The do's and don'ts of minimum viable product marketing"
date: 2026-05-10
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
    - joe-martin
featuredImage: >-
    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-blog-image.png
featuredImageType: full
category: Inside PostHog
tags:
    - Marketing
    - Guides
---

PostHog is a much bigger company than it was when I joined in 2022 as the first product marketer – but while we've more than 10xed in company size since 2022, the product marketing team has only quadrupled. That's deliberate, and it's possible because we've built a marketing culture that reflects our engineering culture: we put emphasis on shipping, [not collaboration](https://newsletter.posthog.com/p/collaboration-sucks).

> Most product marketing advice tells you the opposite. Plan your messaging pillars in advance. Centralize the team around one approach. Block out a launch week. Build the full deck before anyone touches the post. I've worked in teams like this and it's eventually what gives PMMs their bad reputation: elaborate Powerpoints that get lobbed over the fence to other teams as high-level requests for them to do the actual work.

Author: ```suggestion
Most product marketing advice tells you the opposite. Plan your messaging pillars in advance. Centralize the team around one approach. Block out a launch week. Build the full deck before anyone touches the post. I've worked in teams like this and it's eventually what gives product marketing managers (PMMs) their bad reputation: elaborate Powerpoints that get lobbed over the fence to other teams as high-level requests for them to do the actual work.
```

> Most product marketing advice tells you the opposite. Plan your messaging pillars in advance. Centralize the team around one approach. Block out a launch week. Build the full deck before anyone touches the post. I've worked in teams like this and it's eventually what gives PMMs their bad reputation: elaborate Powerpoints that get lobbed over the fence to other teams as high-level requests for them to do the actual work.

Author: ```suggestion
Most product marketing advice tells you the opposite. Plan your messaging pillars in advance. Centralize the team around one approach. Block out a launch week. Build the full deck before anyone touches the post. I've worked in teams like this and it's eventually what gives PMMs their bad reputation: creators of elaborate Powerpoints that get lobbed over the fence to other teams as high-level requests for them to do the actual work.
```

Here's how minimum viable product marketing shapes how we plan all of those product launches, structured not as yet-another-marketing-framework, but as functional advice for people who'd rather spend most of their time building. 

## What we focus on

There's no central framework that defines how we think about products before we launch them. The starting point is tactical: what are we actually going to do?

At PostHog, that means a launch checklist. We have a public template for it – a recent example of it in action lives in the [requests-for-comments-public RFC](https://github.com/PostHog/requests-for-comments-public/issues/513). The template is a guideline, not a rule — [they get tweaked every time](https://github.com/PostHog/requests-for-comments-public/issues/313). There are items we feel strongly about, but we treat every launch differently because every launch is different. The checklist is a starting point, not a destination.

Here are the things I feel most strongly about every team doing.

> **Write the announcement.** Your announcement may hit multiple channels, like [blogposts](/blog/session-replay-pricing) or [social posts](https://x.com/posthog/status/2051701057667871112), but personally I always start with the email. It's likely the most intrusive comm you'll write, so it should be the shortest – and that constraint forces you to decide what the story you're actually telling is. If the email is interesting, the blog post will be. If it isn't, neither will be.

Reviewer: ```suggestion
**Write the announcement first.** Your announcement may hit multiple channels, like [blogposts](/blog/session-replay-pricing) or [social posts](https://x.com/posthog/status/2051701057667871112), but personally I always start with the email. It's likely the most intrusive comm you'll write, so it should be the shortest – and that constraint forces you to decide what the story you're actually telling is. If the email is interesting, the blog post will be. If it isn't, neither will be. (This is the hard bit, so I've written more about how I personally approach this below. Spoiler: It's basically creative writing.)
```

> This is the hard bit, so I've written more about how I personally approach this below. Spoiler: It's basically creative writing. 

Reviewer: ```suggestion
```

Still, there are things I feel strongly about us doing for every launch...

**Write an announcement.** Your announcement may span multiple channels, like [blogposts](/blog/session-replay-pricing) or [social posts](https://x.com/posthog/status/2051701057667871112). Personally, I  start with the email because it's the most intrusive to the user. That means it should be the shortest and that forces clarity. If the email is clear, the blog post will be. If it isn't, neither will be.

This is the hard bit, so I've written more about how I personally approach this below. Spoiler: It's basically just creative writing. 

**Write the title first.** If you can't write the title, you don't know what you're announcing. The title is the only part of the post most readers will see, so it needs to be right. I find it helps not to spend time perfecting it: just get a vomit draft out quickly and go from there. Writing is rewriting, after all. 

> **Focus on conflict.** Conflict takes many forms, but it's ultimately at the root of every story worth telling. Maybe the conflict is the problem your product solves, or the competitors you're standing against. Maybe you're standing on principle or disrupting an industry. When we cut session replay pricing by more than 50%, we titled the post '[We've decided to make less money](/blog/session-replay-pricing)' That's the conflict (us vs. shareholders) and the change (we're cutting prices) in eight words.

Reviewer: ```suggestion
**Focus on conflict.** Conflict takes many forms, but it's ultimately at the root of every story worth telling. Maybe the conflict is the problem your product solves, or the competitors you're standing against. Maybe you're standing on principle or disrupting an industry. When we cut session replay pricing by more than 50%, we titled the post "[We've decided to make less money.](/blog/session-replay-pricing)" That's the conflict (us vs. shareholders) and the change (we're cutting prices) in eight words.
```

**Creating consensus.** Barring some [notable exceptions](/community/profiles/30208), businesspeople are not experienced editors or marketers and do not need to be treated as such. Feedback should be listened to, but not every suggestion should be taken — [especially if it suggests a solution](https://www.goodreads.com/quotes/477087-remember-when-people-tell-you-something-s-wrong-or-doesn-t-work).  

## How can this possibly scale?

PostHog's product marketing team is still small, but so far this has scaled well. Everyone has freedom to approach things in their own style and in this way we know we're always using the right tool for the job. 

Is it chaotic? Sure. 

Will it still work in another five years? I don't know! 

What I _do_ know is that product marketing is full of thought-leaders who want to inflate their importance by making their work more complicated and mysterious than it needs to be. The reality is that the job is basically just storytelling, and storytelling can be messy. 

> That's a good thing. It means you don't need messaging matrices, fancy frameworks, or page-long personas. All you really need is to know [Orwell's sixth rule of writing](https://sites.duke.edu/scientificwriting/orwells-6-rules/) and apply it vigorously.

Author: ```suggestion
That's a good thing. It means you don't need messaging matrices, fancy frameworks, or page-long personas. All you really need is to know [Orwell's sixth rule of writing](https://sites.duke.edu/scientificwriting/orwells-6-rules/) and apply it vigorously. Even here. 
```

> That's a good thing. It means you don't need messaging matrices, fancy frameworks, or page-long personas. All you really need is to know [Orwell's sixth rule of writing](https://sites.duke.edu/scientificwriting/orwells-6-rules/) and apply it vigorously. Even here. 

Reviewer: ```suggestion
That's a good thing. It means all you really need is to know [Orwell's sixth rule of writing](https://sites.duke.edu/scientificwriting/orwells-6-rules/) and apply it vigorously. No fancy messaging matrices, fancy frameworks, or page-long personas needed. 
```

(to make sure the article's ends on a note that recaps this was about "you dont need a fancy framework")
